### PR TITLE
Add prices for untradable items

### PR DESCRIPTION
**Pull request by Adam-**

### CODE REVIEW

**Comment by iguaan** [comment]
I might've missed them, but I didn't notice infernal axe/pickaxe/harpoon (charged and uncharged variants) being listed.

File: runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java [hunk 1]
```diff
@@ -0,0 +1,244 @@
...
+		return mapping;
+	}
+
+	/**
+	 * Map an item from its untradeable version to its tradeable version
+	 *
+	 * @param itemId
+	 * @return
+	 */
+	public static int mapOne(int itemId)
```
Comment (deathbeam) [inline runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: This should be renamed to mapFirst so it properly indicates what the method actually does.

**Comment by deathbeam** [comment]
PRs welcome @iguaan this is just base, do not includes everything out of the box :d

File: runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+	ITEM_TORAGS_PLATEBODY(TORAGS_PLATEBODY, TORAGS_PLATEBODY_25, TORAGS_PLATEBODY_50, TORAGS_PLATEBODY_75, TORAGS_PLATEBODY_100),
+	ITEM_TORAGS_PLATELEGS(TORAGS_PLATELEGS, TORAGS_PLATELEGS_25, TORAGS_PLATELEGS_50, TORAGS_PLATELEGS_75, TORAGS_PLATELEGS_100),
+	ITEM_TORAGS_HAMMERS(TORAGS_HAMMERS, TORAGS_HAMMERS_25, TORAGS_HAMMERS_50, TORAGS_HAMMERS_75, TORAGS_HAMMERS_100),
+	ITEM_VERACS_HELM(VERACS_HELM, VERACS_HELM_25, VERACS_HELM_50, VERACS_HELM_75, VERACS_HELM_100),
+	ITEM_VERACS_BRASSARD(VERACS_BRASSARD, VERACS_BRASSARD_25, VERACS_BRASSARD_50, VERACS_BRASSARD_75, VERACS_BRASSARD_100),
+	ITEM_VERACS_PLATESKIRT(VERACS_PLATESKIRT, VERACS_PLATESKIRT_25, VERACS_PLATESKIRT_50, VERACS_PLATESKIRT_75, VERACS_PLATESKIRT_100),
+	ITEM_VERACS_FLAIL(VERACS_FLAIL, VERACS_FLAIL_25, VERACS_FLAIL_50, VERACS_FLAIL_75, VERACS_FLAIL_100),
+
+	// Dragon equipment ornament kits
+	ITEM_DRAGON_SCIMITAR(DRAGON_SCIMITAR, DRAGON_SCIMITAR_OR),
```
Comment (Nightfirecat) [inline runelite-client/src/main/java/net/runelite/client/game/ItemMapping.java]: For dragon equipment, godswords, jewellery, does the price lookup find the combined item + ornament kit price, or just the item price? Unlike a lot of the other items, the ornament kits can be removed from these items.

**Pull request merged**